File: ExcelDivider/src/main/java/ru/tsystems/divider/dao/api/FeatureDao.java
package ru.tsystems.divider.dao.api;

import ru.tsystems.divider.entity.Feature;

public interface FeatureDao extends GeneralDao<Feature>{

    /**
     * Get entity by param if it exist, return null if not.
     * @param param     String with param.
     * @return          Entity if it exist either null.
     */
    Feature getByParam(String param, String nature);

}

File: ExcelDivider/src/main/kotlin/ru/tsystems/divider/dao/api/FeatureDao.kt
package ru.tsystems.divider.dao.api

import ru.tsystems.divider.entity.Feature

interface FeatureDao : GeneralDao<Feature> {

    /**
     * Get entity by param if it exist, return null if not.
     * @param param     String with param.
     * @return          Entity if it exist either null.
     */
    fun getByParam(param: String, nature: String): Feature?

}